src/app/components/Slideout/SlideoutButton.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <!-- Centers images -->
+  <button
+    class='border-0'
+     @click="$emit('input')"
+     :aria-label="`Open the ${name} tab of the slideout`"
```

Comment:
As mentioned in the other usage of aria-label, maybe better to be just `${name}`, or have it configurable by the specific component. It should be a label (usually 1-2 words), not a description